transformer/kubernetes/buildconfigtransformer.go

```diff
@@ -60,13 +49,23 @@ type BuildConfigYamlConfig struct {
 	OutputPath string `yaml:"outputPath"`
 }
 
+const (
+	defaultBuildConfigYamlsOutputPath = common.DeployDir + string(os.PathSeparator) + common.CICDDir + string(os.PathSeparator) + "buildconfig"
```

Comment:
lets use filepath.Join

Reply:
if we use a function then we have to change it to `var`.